pycbc/workflow/segment.py

```diff
@@ -1129,6 +1129,7 @@ def get_analyzable_segments(workflow, out_dir, tags=[]):
         curr_file = OutSegFile(ifo, 'SCIENCE_OK', workflow.analysis_time,
                                curr_url, segment_list=sci_segs[ifo],
                                tags=tags)
+        curr_file.PFN(seg_ok_path, 'local')
```

Comment:
Ian,

The only reason I didn't was that this was done elsewhere many times, and
it is a failure to add the path twice, I believe. Without the time to check
and fix everywhere else, I wanted to make the minimal change. In principle,
I would actually like the toSegmentXml method to be replaced by the
segments_to_file function, which does this also. Add this to the list of
segment related cleanup.

On Mon, Sep 28, 2015 at 3:54 PM, spxiwh <notifications@github.com> wrote:

> In pycbc/workflow/segment.py
> <https://github.com/ligo-cbc/pycbc/pull/387#discussion_r40553916>:
>
> > @@ -1129,6 +1129,7 @@ def get_analyzable_segments(workflow, out_dir, tags=[]):
> >          curr_file = OutSegFile(ifo, 'SCIENCE_OK', workflow.analysis_time,
> >                                 curr_url, segment_list=sci_segs[ifo],
> >                                 tags=tags)
> > +        curr_file.PFN(seg_ok_path, 'local')
>
> I would really prefer it if you could move this into the toSegmentXml
> method in core.py
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ligo-cbc/pycbc/pull/387/files#r40553916>.
>